fog/test-client/src/test_client.rs

```diff
@@ -224,7 +224,7 @@ impl TestClient {
         &self,
         source_client: &mut Client,
         target_client: &mut Client,
-    ) -> Result<(Tx, u64), TestClientError> {
+    ) -> Result<(Tx, u64, u64), TestClientError> {
```

Comment:
Can we add some docu explaining what this return value is?

Reply:
Done.